### ISSUE TITLE
Inner and Outter Class Prefixes inverted

### ISSUE THREAD

**Issue by klquast** [subarroca/ng-password-strength#14]
When the password strength is low, the css classes are inverted. This is what the html looks like when the strength is low: 
```html
<div class="progress  progress-bar-alert">
    <div class="progress-bar danger"></div>
</div>
```
vs what the html looks like when the strength is medium or high:
```html
<div class="progress  warning">
    <div class="progress-bar progress-bar-warning"></div>
</div>
<div class="progress  success">
    <div class="progress-bar progress-bar-success"></div>
</div>
```
Looks like you're setting **inner** to scope.**outter**ClassPrefix and **outter** to scope.**inner**ClassPrefix [here](https://github.com/subarroca/ng-password-strength/blob/master/app/scripts/directives/ng-password-strength.js#L224).

Also, the alert class is being used incorrectly here for Bootstrap mode (looks like the "alert" class in Foundation is equivalently to the "danger" class in Bootstrap). Right now, because the outer class is set to "alert", the progress bar is blue instead of red when the password strength is low. Both the inner and outer class should be set to prefix + "danger" for Bootstrap mode.

**Comment by mikedevita** [comment]
is there a workaround for this?

**Comment by subarroca** [comment]
I'll try to make the changes this week though I'm quite busy at the moment. Or else you can change it and do a pull request.

**Comment by intellix** [comment]
Fixed in #16

**Comment by bdkent** [comment]
Any idea if this will make it into a release? This PR does not appear in the latest in bower (0.2.1).

**Comment by subarroca** [comment]
I should probably make a release. I'll have a look at all pull requests and make a new release.

**Comment by intellix** [comment]
Just a reminder that this can be closed as it was fixed in my PR #16
